CONTRIBUTING.md

```diff
@@ -32,3 +34,48 @@ the code they are contributing to the project according to the
 
 As stated in [QUALITY.md](QUALITY.md), all contributions to the project must increase line code coverage.
 Because of this, contributors are asked to locally run a coverage assessment that ensures that code coverage has increased when compared to the latest execution of the [nightly coverage CI job](http://jenkins.eprosima.com:8080/job/nightly_fastdds_coverage_linux/).
+
+## Issues and Support
+
+*eProsima Fast DDS* developers welcome all contributions, and we will be grateful if you follow the guidelines below while contributing in this project.
+This project classifies user issues in the following categories.
+Please, help us giving you a better support by opening your issue in the corresponding category:
+
+### Issues
+
+Fast DDS issues are understood mostly as bug reports and may be opened by anyone [here](https://github.com/eProsima/Fast-DDS/issues/new/choose).
+Consequently, this section handles malfunctions in the current documented behavior of the library or non-compliances to the DDS specification.
+If you are unsure on whether your experienced behavior falls into this category, please open a ticket in the [Support](#support) discussion forum and, if it is a malfunction, an issue will be opened on your behalf with the provided report.
+
+An issue template is provided and it is really appreciated if all related information is provided so the issue may be reproduced.
+Otherwise, more information could be required in order to reproduce and solve the issue.
+If this is the case, the issue will be labeled with `need more info`.
+Please, be advised that in case that no response is received within a month, the issue would be closed due to inactivity.
+The issue may be reopened if the required information is provided.
+
+### Feature request
+
+Feature requests and improvements suggestions should be opened in the Ideas [discussion forum](https://github.com/eProsima/Fast-DDS/discussions/new).
```

Comment:
Is there a way to automatically select the `Ideas` category on the link?

Same for similar links below

Reply:
Right now it does not seem to be possible. When a new discussion is opened, the category is left empty by Github. If the user tries to open a discussion without setting the category, Github reminds the user to select one.